src/modules/feeds/createPostModal/useCreatePostModal.tsx

```diff
@@ -57,7 +57,7 @@ export const useCreatePostModal = (
     setValue,
     watch,
     reset,
-  } = useForm<Form>({
+  } = useForm({
```

Comment:
please do not remove the type, it helps us for using and achieving data in form easier.

Reply:
@satanmourner Can you please handle this files: useCreatePostModal and CreatePostModal.types? Since you have worked on that, you are more familiar with the logic. I'm afraid I change something that impacts the functionality